.github/workflows/develop_commit_notification.yml

```diff
@@ -1,15 +1,18 @@
 name: Broadcast 'push to develop' event to our release-scripts repo
 on:
-  push:
-    branches:
-      - develop
+  status
 
 jobs:
   send_notification:
     name: Send notification
     runs-on: ubuntu-22.04
-    if: github.repository == 'oppia/oppia'
+    if: ${{ (github.repository == 'oppia/oppia') && (github.event.state == 'success') }}
```

Comment:
Just to check, what is tracked by this condition? It might be worth adding a comment to explain.

- Will it include success on PR runs (before they're merged)?
- Let's say someone reruns a really old commit on develop, and the CI tests pass for that rerun. Will this trigger? (Should perhaps guard against that if the old commit is older than the latest deployed version.)

Reply:
Ah sorry, I see the second point is guarded against in the other PR. Still, I'd suggest adding a comment here to explain what we're catching and which events will be filtered out subsequently (and where the filtering is done).

Reply:
Added a comment for both points.

For point (1), I think it should trigger only for develop branch (and not PRs) because of this statement: "This event will only trigger a workflow run if the workflow file is on the default branch." [here](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#status).

Reply:
Mmm, that talks about the workflow file being on the default branch, not the status change being on the default branch.

Is there a way we can check this? Or do we just merge this and see what happens? (If the latter, what's the plan if it ends up firing for PRs?)

Reply:
I tried this out on my fork and turns out the status event gets triggered even for updates on different branches. Let me see if there is a way to narrow it down so that the workflow executes only for status change on develop. If not, I will check if there is another way to trigger a build _after_ checks have run.

Reply:
Update: I've updated this such that the "build and deploy" is triggered after the tests have run by moving this logic to "full_stack_tests.yml".